pkg/server/catalog/catalog.go

```diff
@@ -36,6 +37,39 @@ import (
 	up_spire "github.com/spiffe/spire/pkg/server/plugin/upstreamca/spire"
 )
 
+var (
+	portedUpstreamCA = map[string]bool{}
+
+	builtIn = []catalog.Plugin{
```

Comment:
I'm not sure about the impact of using this variable. Before this change, the array was returned inside the BuiltIns function, which in turn called each BuiltIn function of each package, so initialization of the array was done every time the BuiltIns function was called. Now, the BuiltIns functions will be called only on initialization time...which might be a good thing, but I just want to confirm that we are all on the same page.

Reply:
I just wanted to make sure that the change in semantics was ok, if it is, then no need to change anything I believe.